### PR TITLE
Add POL Major

### PR DESCRIPTION
**Pull request by danielleung27**
Add pol major

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+      "completed_by_semester": 6
+    },
+    {
+      "name": "Senior Thesis",
+      "min_needed": 0,
+      "explanation": "During the senior year, each student writes a thesis. The senior thesis (link is external) normally is written on a topic within a student's primary field. The department encourages students to use the summer between junior and senior year for work on the senior thesis.",
+      "completed_by_semester": 8
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "completed_by_semester": 8,
      "no_req": null
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+      "completed_by_semester": 6
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "completed_by_semester": 6,
      "no_req": null
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Junior Seminar",
+      "max_counted": 1,
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "max_counted": null,
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+      "min_needed": 1,
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "min_needed": null,
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+      "completed_by_semester": 6
+    },
+    {
+      "name": "Senior Thesis",
+      "min_needed": 0,
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "min_needed": null,
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+      "explanation": "During the senior year, each student writes a thesis. The senior thesis (link is external) normally is written on a topic within a student's primary field. The department encourages students to use the summer between junior and senior year for work on the senior thesis.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]:
```suggestion
      "explanation": "During the senior year, each student writes a thesis. The senior thesis normally is written on a topic within a student's primary field. The department encourages students to use the summer between junior and senior year for work on the senior thesis.",
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
...ternal). Junior Workshops will meet every other week of the Fall teaching period and on alternating weeks, students are required to view online plenary session video modules and attend a 50-minute discussion section. The plenary component provides common instruction in research procedures and techniques. In their Fall Junior Workshops, juniors will complete a series of assignments building up to a research prospectus. [NOTE: Junior Workshops (link is external) are not considered courses and do not count toward satisfying departmental course requirements.] In the spring semester, juniors will complete a Junior Paper under the supervision of an independent work adviser. Students must achieve at least a “D” on each of their two semesters of junior independent work as well as an average of a “C” or above across the two. If the average of the two is below “C” or one of the two receives an “F”, a second Junior Paper is required (with a grade that brings the average of the Fall JIW and the Summer JP grades to at least a “C” or better). This is a prerequisite for beginning the senior year. For purposes of this requirement, the grades before the application of any late penalties are used.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]: Remove any appearances of `(link is external)`
```suggestion
      "explanation": "Students are required to complete two semesters of junior independent work. Junior Workshops will meet every other week of the Fall teaching period and on alternating weeks, students are required to view online plenary session video modules and attend a 50-minute discussion section. The plenary component provides common instruction in research procedures and techniques. In their Fall Junior Workshops, juniors will complete a series of assignments building up to a research prospectus. [NOTE: Junior Workshops are not considered courses and do not count toward satisfying departmental course requirements.] In the spring semester, juniors will complete a Junior Paper under the supervision of an independent work adviser. Students must achieve at least a “D” on each of their two semesters of junior independent work as well as an average of a “C” or above across the two. If the average of the two is below “C” or one of the two receives an “F”, a second Junior Paper is required (with a grade that brings the average of the Fall JIW and the Summer JP grades to at least a “C” or better). This is a prerequisite for beginning the senior year. For purposes of this requirement, the grades before the application of any late penalties are used.",
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+        "PHI 201",
+        "SOC 404",
+        "WWS 200",
+        "WWS 332"
+      ]
+    },
+    {
+      "name": "Track",
+      "max_counted": 1,
+      "min_needed": null,
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json:195]: Isn't at least one track required?
```suggestion
      "min_needed": 1,
```

File: tigerpath/majors_and_certificates/majors/POL_2018.json [hunk 9]
```diff
@@ -0,0 +1,438 @@
...
+            "POL 453"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Analytic Requirement",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "The department maintains a list of politics courses that have an emphasis on methodological tools for research in political science. Concentrators are required to take a course to fulfill the analytic requirement (link is external), normally no later than the first term of their junior year. The courses used to fulfill the analytic requirement cannot be used to fulfill primary-, secondary-, or third-field requirements. The analytic requirement may be satisfied by POL 341, POL 345/SOC 305, POL 346, or POL 347. We will also accept ANT 300A, ANT 301A, ECO 202, ECO 302, ECO 312, ORF 245, PHI 201, SOC 404, WWS 200, or WWS 332.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/POL_2018.json]: Remove `(link is external)`
```suggestion
      "explanation": "The department maintains a list of politics courses that have an emphasis on methodological tools for research in political science. Concentrators are required to take a course to fulfill the analytic requirement, normally no later than the first term of their junior year. The courses used to fulfill the analytic requirement cannot be used to fulfill primary-, secondary-, or third-field requirements. The analytic requirement may be satisfied by POL 341, POL 345/SOC 305, POL 346, or POL 347. We will also accept ANT 300A, ANT 301A, ECO 202, ECO 302, ECO 312, ORF 245, PHI 201, SOC 404, WWS 200, or WWS 332.",
```

**Review by bnehoran — CHANGES_REQUESTED** [review]

File: tigerpath/majors_and_certificates/majors/POL_2018.json (hunk 8 above)
Comment (danielleung27) [reply tigerpath/majors_and_certificates/majors/POL_2018.json:195]: I think they're optional. https://ua.princeton.edu/academic-units/department-department-politics#

**Comment by danielleung27** [comment]
Major is working and tested, after @194 is merged, it should work.

File: tigerpath/majors_and_certificates/majors/POL_2018.json (hunk 8 above)
Comment (bnehoran) [reply tigerpath/majors_and_certificates/majors/POL_2018.json:195]: I see

**Review by bnehoran — APPROVED** [review]

**Pull request merged**